packages/compass-indexes/src/stores/store.js

```diff
@@ -96,8 +95,7 @@ const configureStore = (options = {}) => {
   }
 
   if (options.isReadonly) {
-    const isReadonlyView = options.isReadonly;
-    store.dispatch(readonlyViewChanged(isReadonlyView));
```

Comment:
We are using `isReadonlyView` within this plugin to identify a view and `isReadonly` was not set anywhere and always defaulted to its `false` value. So I removed it in this commit.

Reply:
It's not always `false`, it's `true` if you connect to ADL for example, we can't just remove it

Reply:
Ah, I see what you mean, the state is always `false`, not the option passed to the `configureStore`. 

Something is super weird going on here. I see that before https://github.com/mongodb-js/compass/pull/3609 this was actually mapping to hadron readonly mode, the UI was updated to read the value from preferences, but the store, in particular `fetchIndexes` method wasn't, I see that you replaced it with reading `isReadonlyView`, but this doesn't seem what the meaning of this value was

Reply:
So it was a bug that was accidentally fixed by not checking for preferences there?

Reply:
Yes it was.